app/store/persistent-state/state-migrations.ts

```diff
@@ -30,9 +30,24 @@ type PersistentState_6 = {
   galoyAuthToken: string
 }
 
-const migrate6ToCurrent = (state: PersistentState_6): Promise<PersistentState> =>
+type PersistentState_7 = {
+  schemaVersion: 7
+  galoyInstance: GaloyInstanceInput
+  galoyAuthToken: string
+  galoyAllAuthTokens: string[]
+}
+
+const migrate7ToCurrent = (state: PersistentState_7): Promise<PersistentState> =>
   Promise.resolve(state)
 
+const migrate6ToCurrent = (state: PersistentState_6): Promise<PersistentState> => {
+  return migrate7ToCurrent({
+    ...state,
+    schemaVersion: 7,
+    galoyAllAuthTokens: [],
```

Comment:
Shouldn't this check if the user has a token preset and add it in the All array?

Reply:
Yes it should be present in the all tokens list